motes.md

```diff
@@ -0,0 +1,42 @@
+* `BLS_WITHDRAWAL_PREFIX`
+  * Why int rather than bytes?
+* `MIN_SEED_LOOKAHEAD`
+  * Is this actually tunable?
+  * If so, what are the reprecussions?
+* `ACTIVATION_EXIT_DELAY`
+  * Reaquaint with purpose
+* AttesterSlashings
+  * `MAX_ATTESTER_SLASHINGS` is 1.
+  * Are there scenarios in which validators can create more effective slashable
+    messages than can be included on chain? For example, Validators split up to
+    create double attestations for checkpoints but different (junk) crosslink
+    data to prevent them from being aggregatable to the fullest
+  * Max is for block size, no?
+* Signature domains
+  * Max 4byte ints
+* `Version` not defined in one of the lists of custom types (2!!). ensure in spec
+* `PendingAttestation`
+  * Don't think `proposer_index` is actually necessary here because effective
+    balance is stable until end of epoch so can do dynamic lookups
```

Comment:
Oh, nice find! We only use `proposer_index` once so this is a nice cleanup I guess.